setup.py

```diff
@@ -1,17 +1,38 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
+from setuptools import setup
+from codecs import open
 
 
-versionfile = open('VERSION', 'r')
-ver = versionfile.read().strip()
-versionfile.close()
+with open('VERSION', encoding='utf-8') as version:
+    ver = version.read().strip()
 
-setup(name='scot',
-      version=ver,
-      description='Source Connectivity Toolbox',
-      author='Martin Billinger',
-      author_email='martin.billinger@tugraz.at',
-      url='https://github.com/scot-dev/scot',
-      packages=['scot', 'scot.eegtopo', 'scot.external'],
-      install_requires=['numpy >=1.7', 'scipy >=0.12'])
+with open('README.md', encoding='utf-8') as readme:
+    long_description = readme.read()
+
+setup(
+    name='scot',
+    version=ver,
+    description='Source Connectivity Toolbox',
+    long_description=long_description,
+    url='https://github.com/scot-dev/scot',
+    author='SCoT Development Team',
+    author_email='martin.billinger@tugraz.at',
+    license='MIT',
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Intended Audience :: Science/Research',
+        'Topic :: Scientific/Engineering',
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.2',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+    ],
+    keywords='source connectivity EEG MEG ICA',
+    packages=['scot', 'scot.eegtopo', 'scot.external'],
+    install_requires=['numpy >=1.7', 'scipy >=0.12'],
```

Comment:
OK. On my Mac, I always used pip to install numpy and scipy and never had any problems. I don't know about Linux though, but you are right, on Linux I would also prefer installing these packages through the distribution's package manager and not via pip.